.travis.yml

```diff
@@ -6,42 +6,21 @@ cache:
 
```

Comment:
The entire file `.travis.yml` should be deleted, since we start to use GitHub Actions with this PR.